src/richie/apps/core/context_processors.py

```diff
@@ -107,6 +108,11 @@ def site_metas(request: HttpRequest):
         context[
             "RICHIE_MINIMUM_COURSE_RUNS_ENROLLMENT_COUNT"
         ] = settings.RICHIE_MINIMUM_COURSE_RUNS_ENROLLMENT_COUNT
+
+    context["MAX_ARCHIVED_COURSE_RUNS"] = getattr(
+        settings, "MAX_ARCHIVED_COURSE_RUNS", MAX_ARCHIVED_COURSE_RUNS
```

Comment:
To distinguish clearly settings concerned by RICHIE we prefix these settings with `RICHIE_`
```suggestion
        settings, "RICHIE_MAX_ARCHIVED_COURSE_RUNS", MAX_ARCHIVED_COURSE_RUNS
```